template/page-uses.html.jinja

```diff
@@ -16,136 +16,492 @@
-        "description": _("Forte offers an unprecedented set of easy-to-use tools and services for game developers to integrate blockchain technology into their games, to unlock new economic and creative opportunities for gamers across the world.")},
-      { "link": "https://gatehub.net/",
-        "id": "gatehub",
-        "name": "Gatehub",
-        "title": _("Wallet and Platforms"),
-        "description": _("GateHub is a platform for the Internet of Value, built on the XRP Ledger protocol. It allows everyone to send, receive, trade and manage any type of assets.")},
-      { "link": "https://www.exodus.io/",
-        "id": "exodus",
-        "name": "Exodus",
-        "title": _("Wallets and Apps"),
-        "description": _("Exodus offers wallets and applications for securing, managing and exchanging crypto.")},
-      { "link": "https://ripple.com/",
-        "id": "ripple",
-        "name": "Ripple",
-        "title": _("On-Demand Liquidity"),
-        "description": _("Ripple powers instant, lower-cost settlement of cross-border payments using XRP to source liquidity on demand. XRP is ideally suited for global payments because it's quicker, less costly, and more scalable than any other digital asset.")},
-      { "link": "https://towolabs.com",
-        "id": "towo",
-        "name": "Towo Labs",
-        "title": _("Infrastructure"),
-        "description": _("Towo Labs was founded in 2019, to develop XRP Ledger and Interledger infrastructures and make non-custodial crypto management easier.")},
-      { "link": "https://raisedinspace.com/",
-        "id": "raisedinspace",
-        "name": "Raised in Space",
-        "title": _("Music"),
-        "description": _("Raised in Space is a music/tech investment group focused on raising the value of music, innovating across the entire value chain of the music industry.")},
-      { "link": "https://xrpl-labs.com/",
-        "id": "xrpl-labs",
-        "name": "XRPL Labs",
-        "title": _("Applications"),
-        "description": _("From cold storage to apps for signing transactions, XRPL Labs is dedicated to building software on the XRP Ledger.")},
-      { "link": "https://xrplorer.com/",
-        "id": "xrplorer",
-        "name": "Xrplorer",
-        "title": _("Security"),
-        "description": _("Xrplorer offers services and tools that help prevent and combat fraudulent activity on the XRPL as well as custom APIs and analytics that supplement the XRPL APIs where they are not enough.")},
-    ] %}
-
-    <div class="row row-cols-1 row-cols-lg-3 card-deck">
-      {% for card in cards %}
-      <a class="card" href="{{card.link}}" target="_blank" id="{{card.id}}">
-        <div class="card-body">
-          <img class="mw-100 mb-3 biz-logo" alt="{{card.name|default(card.id)}}">
-          <h4 class="card-title h5">{{card.title}}</h4>
-          <p class="card-text">{{card.description}}</p>
+    <a class="btn  d-block d-lg-none" data-toggle="modal" data-target="#categoryFilterModal">
```

Comment:
It seems less than ideal to have the category filter stuff repeated in the code for both the modal and the sidebar. Even if the resulting HTML has to have both instances, it might probably better for maintainability if they shared some code to generate them (a macro or something?). Maybe that's more work than it's worth.

Reply:
since sidebar and mobile filter menu are slightly different they needed to be in sync with changes.  You are correct there has to be a better way. I can re work if time is allotted.